packages/apps-config/src/api/spec/crab.ts

```diff
@@ -0,0 +1,230 @@
+// Copyright 2017-2020 @polkadot/apps-config authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+export default {
+  '__[frame.system]__': {},
```

Comment:
These are weird. Would really like to track where they come from. (All good, just seems like something that should not be decorated by the API at all)

Reply:
Was happy with them if needed to operate, just weird if they do get decorated by the API (that would point to some issue in that layer).